app/src/main/java/org/hackillinois/android/database/entity/Project.kt

```diff
@@ -0,0 +1,28 @@
+package org.hackillinois.android.database.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.TypeConverters
+import org.hackillinois.android.database.Converters
+
+@Entity(tableName = "projects")
+@TypeConverters(Converters::class)
+data class ProjectModel(
+    @PrimaryKey val id: String,
+    val name: String,
+    val description: String,
+    val mentors: List<String>,
+    val room: String,
+    val tags: List<String>,
+    val number: Int
+)
+
+fun getMentorsString(project: ProjectModel): String {
```

Comment:
Can you put this function inside the class? You'll have to add a left brace on line 18 and then a right brace after this.

Reply:
And also remove the argument and just use `this` instead